content/docs/spec.md

```diff
@@ -151,7 +153,8 @@ The C1 and S1 packets are 1536 octets long, consisting of the following fields:
 |                             ...                               |
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
                         C1 and S1 bits
```

Comment:
should `C1 and S1 bits` be <figcaption> for consistency?

Reply:
Yup, agree. (Same issue as https://github.com/veriskope/rtmp-website/pull/18#discussion_r371599109)